src/universal/common/components/AppHeader/AppHeader.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import {Link} from 'react-router';
+import {AppLogo} from 'universal/common/components';
+
+export default class AppHeader extends React.Component<any, any> {
+  state = {
+    isScrolled: false
```

Comment:
We don't need the `isScrolled` logic here. This was supposed to set a state variable when the window was scrolled, to be able to make the header smaller when scrolling.